src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheckTest.java

```diff
@@ -49,7 +49,6 @@ public void testDefault() throws Exception {
             "266:1: " + getCheckMessage(MSG_KEY, ";"),
             "270:16: " + getCheckMessage(MSG_KEY, ";"),
             "284:1: " + getCheckMessage(MSG_KEY, ";"),
-            "287:62: " + getCheckMessage(MSG_KEY, "..."),
```

Comment:
please add to google style IT Inputs bunch of code with all possible combinations of `...` and annotations.

Reply:
done